fake_switches/netconf/__init__.py

```diff
@@ -125,6 +125,11 @@ def __init__(self, vlan, interface, unit):
         super(UnknownVlan, self).__init__("No vlan matches vlan tag %s for interface %s.%s" % (vlan, interface, unit))
 
 
+class UnknownInterface(NetconfError):
+    def __init__(self, interface):
+        super(UnknownInterface, self).__init__("Interface {} not found".format(interface))
```

Comment:
Yes, we were just looking at the responses. We will be correcting it. There are actually two different error messages that will come up based on input. We will have to re-implement our exception handler based on the data we receive.